scripts/wheels/linux/build.sh

```diff
@@ -5,10 +5,21 @@ export KRATOS_VERSION="9.0.0"
 
 BASE_LD_LIBRARY_PATH=$LD_LIBRARY_PATH
 export KRATOS_ROOT="/workspace/kratos/Kratos"
+export KRATOS_BUILD_TYPE="Custom"
```

Comment:
i suggest to allow calling the docker image with this as env variable (or command option like the branch name) This would allow to run these tests with Custom mode for faster builds)